app/controllers/concerns/authentication.rb

```diff
@@ -60,7 +60,7 @@ def rbac_allowed?
     return valid_cert_auth? if identity_header.cert_based?
 
     @rbac_api ||= ::RbacApi.new(raw_identity_header)
-    @rbac_api.check_user
+    @rbac_api.full_access?
   end
```

Comment:
This should be replaced with `current_user.something?` to actually leverage what you implemented.